torchprime/torch_xla_models/llama/model.py

```diff
@@ -203,10 +202,11 @@ def __init__(self, config: DictConfig, layer_idx: int | None = None):
     self._init_rope()
 
   def _init_rope(self):
+    scaling = self.config.get("rope_scaling", None)
+    if scaling is not None:
+      scaling = RopeScaling.from_dict(scaling)  # type: ignore
```

Comment:
nit: Is it possible to initialize like to avoid dataclasses-json dependency:

```
scaling = RopeScaling(**scaling)
```
Ref: https://stackoverflow.com/questions/68417319/initialize-python-dataclass-from-dictionary